Sources/Core/JavaModelRenderer.swift

```diff
@@ -213,6 +213,15 @@ public struct JavaModelRenderer: JavaFileRenderer {
 
     // MARK: - TypeAdapter
 
+    func typeAdapterVariableNameForType(_ type: String) -> String {
+        return (type
+            .replacingOccurrences(of: "<", with: "_")
```

Comment:
Can we refactor this logic as an extension on String? I think we do this in a couple of places throughout the codebase to convert a type to a valid variable name

Reply:
added `replacingNonAlphaNumericsWith`